cwf/gateway/services/uesim/test_init/test_service_init.go

```diff
@@ -26,9 +26,9 @@ import (
 
 func StartTestService(t *testing.T) {
 	factory := test_utils.NewSQLBlobstore(t, "uesim_test_service_blobstore")
-	srv, lis := test_utils.NewTestService(t, registry.ModuleName, registry.UeSim)
+	srv, lis, _ := test_utils.NewTestService(t, registry.ModuleName, registry.UeSim)
```

Comment:
So this PR introduces a lot of incidental changes due to changing the signature of NewTestService etc. Normally, we'd stack those changes in a lower diff, but since github makes it so painful I'm fine w lumping everything together